source/accessibility-statement.html.erb

```diff
@@ -48,31 +69,67 @@ title: Accessibility statement for GOV.UK Pay
+
+      <h2 class="govuk-heading-m">How we tested this service</h2>
+      <p class="govuk-body">The service was last tested on 6 August 2020 and was checked for compliance with WCAG 2.1 AA. The test was carried out by the Digital Accessibility Centre.</p>
+      <p class="govuk-body">You can contact us on <a class="govuk-link" href="mailto:govuk-pay-support@digital.cabinet-office.gov.uk">govuk-pay-support@digital.cabinet-office.gov.uk</a> to receive a copy of the full accessibility audit report.</p>
+      <p class="govuk-body">We tested user journeys for both paying users and government service teams, including:</p>
       <ul class="govuk-list govuk-list--bullet">
-        <li>our main website platform, available at <a class="govuk-link" href="https://www.payments.service.gov.uk">https://www.payments.service.gov.uk</a></li>
-        <li>our payments account platform, available at <a class="govuk-link" href="https://selfservice.payments.service.gov.uk">https://selfservice.payments.service.gov.uk</a></li>
-        <li>our credit card payment platform at https://card.payments.service.gov.uk</li>
-        <li>our direct debit payments platform at https://directdebit.payments.service.gov.uk</li>
-        <li>our payment links platform at https://products.payments.service.gov.uk</li>
+        <li>setting up a test account</li>
+        <li>making a demo payment</li>
+        <li>setting up a payment link</li>
+        <li>requesting a live account</li>
+        <li>adding a team member</li>
+        <li>creating a prototype link</li>
+        <li>logging in to an account</li>
+        <li>making a card payment</li>
+        <li>making a card payment with Apple Pay and Google Pay</li>
       </ul>
+      <p class="govuk-body">We previously tested this service for compliance with WCAG 2.1 AA on 5 September 2019.</p>
 
       <h2 class="govuk-heading-m">What we’re doing to improve accessibility</h2>
-      <p class="govuk-body">We know that if you take too long to submit information on a form, you will not receive a warning message or be able to extend your time.</p>
-      <p class="govuk-body">We conducted an accessibility review in 2016 and found that setting a 90-minute timeout  was more helpful for users with accessibility needs than a warning message. However we plan to do more research on this from October 2019.</p>
-      <p class="govuk-body">This page was prepared on 19 September 2019. It was last updated on 19 September 2019.</p>
+      <p class="govuk-body">Our <a class="govuk-link "href="https://www.payments.service.gov.uk/roadmap/">roadmap</a> includes information about how and when we plan to improve accessibility on this service.</p>
```

Comment:
Missing space before`href`. Also this link should be relative: `href="/roadmap/"`.